core/debugger/remote_debugger.cpp

```diff
@@ -62,11 +62,17 @@ class RemoteDebugger::PerformanceProfiler : public EngineProfiler {
 		last_perf_time = pt;
 
 		Array custom_monitor_names = performance->call("get_custom_monitor_names");
+		Array custom_monitor_types = performance->call("get_custom_monitor_types");
+
+		Array custom_monitor_data;
+		custom_monitor_data.clear();
+		custom_monitor_data.append_array(custom_monitor_names);
+		custom_monitor_data.append_array(custom_monitor_types);
```

Comment:
Why not have the two arrays be elements in the array? Like:

```suggestion
		custom_monitor_data.push_back(custom_monitor_names);
		custom_monitor_data.push_back(custom_monitor_types);
```

(Also, I don't think you need the `.clear()` - it should already be empty.)